tests/helpers.py

```diff
@@ -33,16 +36,11 @@ def create_unique_resource_name(resource_description, resource_type):
     return f"{resource_type}-{resource_description[:23]}-{current_date_time[:10]}"
 
 
-def create_resource(
-    desired_status=constants.STATUS_AVAILABLE, wait=True, **kwargs
-):
+def create_resource(**kwargs):
```

Comment:
please ensure all calls to his func has changed as you removed the parameters it gets